django/contrib/admin/utils.py

```diff
@@ -304,9 +304,14 @@ def _get_non_gfk_field(opts, name):
     """
     For historical reasons, the admin app relies on GenericForeignKeys as being
     "not found" by get_field(). This could likely be cleaned up.
+
+    Reverse relations should also be excluded as these aren't attributes of the
+    model (rather something like `foo_set`).
     """
     field = opts.get_field(name)
-    if field.is_relation and field.many_to_one and not field.related_model:
```

Comment:
@timgraham The changes look perfect. I was wondering, isn't is_relation a bit redundant in this case?